test/core/simd/meta/simd_integer_op.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+"""Common integer value operations"""
+
+from simd_lane_value import LaneValue
+
+class IntegerSimpleOp:
+    """Common integer simple ops:
+        min_s, min_u, max_s, max_u
+    """
+
+    @staticmethod
+    def binary_op(op: str, p1: str, p2: str, lane_width: int) -> str:
+        """Binary operation on p1 and p2 with the operation specified by op
+
+        :param op: min_s, min_u, max_s, max_u
+        :param p1: integer value in hex
+        :param p2: integer value in hex
+        :lane_width: bit number of each lane in SIMD v128
+        :return:
+        """
+        if '0x' in p1:
+            base1 = 16
+        else:
+            base1 = 10
```

Comment:
It looks like the parameters are allowed to be base-10, but the doc comment specifies that they must be hex. One of these should be updated.